spid_cie_oidc/onboarding/tests/test_04_authn_response.py

```diff
@@ -0,0 +1,54 @@
+from multiprocessing import AuthenticationError
```

Comment:
I don't know who have suggested multiprocessing to you and @dezhizhang1985 :')

remove this import

If you need an AuthenticationError exception please define it in
https://github.com/peppelinux/spid-cie-oidc-django/blob/main/spid_cie_oidc/entity/exceptions.py

or in an equivalent file in `onboarding`